besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -838,7 +839,7 @@ protected BesuCommand(
    * @param executionExceptionHandler Exception handler for business logic
    * @param in Standard input stream
    * @param args arguments to Besu command
-   * @return success or failure exit code.
+   * @return present or failure exit code.
```

Comment:
this PR doesn't change what BesuCommand.parse returns

Reply:
```suggestion
   * @return success or failure exit code.
```

Reply:
actually this whole comment can be deleted, there's 2 javadoc blocks in a row that are almost identical

Reply:
sorry - that was an intellij refactor